### ISSUE TITLE
Lint for excessive comma in scss

### ISSUE THREAD

**Issue by johannesjo** [stylelint/stylelint#2457]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

I recently discovered an issue, where the linter was not throwing an error for when there is an excessive comma right before the opening curly brace after a selector.

> Which rule, if any, is this issue related to?

Non that I know of.

> What CSS is needed to reproduce this issue?
```scss
.XXXX,
.XXXXXX, {
  display: block;
}
```

> What stylelint configuration is needed to reproduce this issue?

Should appear regardless of config.

> Which version of stylelint are you using?

`7.9.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Running stylelint via node api.

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No.

> What did you expect to happen?
Stylelint should throw an error.

> What actually happened (e.g. what warnings or errors you are getting)?

Stylelint doesn't throw an error.

**Comment by evilebottnawi** [comment]
@johannesjo thanks for using `stylelint`! For linting `scss` syntax use https://github.com/kristerkari/stylelint-scss and create issue here :+1:

**Comment by hudochenkov** [comment]
Try [`stylelint-csstree-validator`](https://github.com/csstree/stylelint-validator) for this purpose.

**Comment by johannesjo** [comment]
Thank you!

**Comment by jeddy3** [comment]
> Try `stylelint-csstree-validator` for this purpose.

Currently, it only validates _property & value pairs_. Validating selectors is [on the roadmap though](https://github.com/csstree/csstree/issues/34).

@johannesjo I'd recommend contributing to `csstree` if you'd like to see this implemented. Or, in the meantime, you can, as suggested in https://github.com/stylelint/stylelint/issues/2155, create a `selector-no-invalid` _plugin_.  You could start with porting [this code](https://github.com/stylelint/stylelint/blob/master/lib/rules/selector-no-empty/index.js) from the now deprecated `selector-no-empty` rule as this should cover your particular use case. Going forward, such a `selector-no-invalid` plugin could be converted to use `csstree`'s  selector validator once it's available.
